.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # Global owner for repo
-* @blink1073 @juliusgeo @ShaneHarvey
+* @mongodb/dbx-python
```

Comment:
I see this error:
> Unknown owner on line 2: make sure the team @mongodb/dbx-python exists, is publicly visible, and has write access to the repository

Maybe it's the "write access" part?

Reply:
Oh, this is a different org, yeah...

Reply:
I'm waiting for the access to be granted before merging.